src/doc/rustc/src/targets/known-issues.md

```diff
@@ -10,4 +10,4 @@ Examples:
 
 | Target-Feature | Issue | Seen on | Description | Details |
 | -------------- | ----- | ------- | ----------- | ------- |
-| `-C target-feature=+soft-float` | Segfaults and ABI mismatches | `x86 and x86-64` | Using soft-floats disables usage of ```xmm``` registers, but parts of ```std``` are compiled without soft-floats and expect parameters to be passed in ```xmm``` registers. This leads to ABI mismatches. | [#63466](https://github.com/rust-lang/rust/issues/63466) |
+| `-C target-feature=+soft-float` <br> or <br> `-C target-feature=-sse` | Segfaults and ABI mismatches | `x86` and `x86-64` | The `x86` and `x86_64` architecture uses SSE registers (aka `xmm`) for floating point operations. Using software emulated floats ("soft-floats") disables usage of `xmm` registers, but parts of `std` (e.g. `std::f32` or `std::f64`) are compiled without soft-floats and expect parameters to be passed in `xmm` registers. This leads to ABI mismatches. <br><br>  Attempting to compile with disabled SSE causes the same error, too. | [#63466](https://github.com/rust-lang/rust/issues/63466) |
```

Comment:
Small editorial suggestion: since the first column is already labeled "Target-Feature" it would make sense (and save some horizontal space) to just write `+soft-float` and `-sse` (not repeating the `-Ctarget-feature=` part).

Reply:
Done. Squashed and rebased commits.